Workbooks/WaasUpdateInsights/SafeguardHolds/SafeguardHolds.workbook

```diff
@@ -0,0 +1,222 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# Update Compliance - Safeguard Holds\n---\nThis Workbook utilizes data from [**Update Compliance**](https://docs.microsoft.com/windows/deployment/update/update-compliance-monitor) to provide a report on active [**safeguard holds**](https://docs.microsoft.com/windows/deployment/update/safeguard-holds) in your environment.\n\nSafeguard holds each have an 8-digit identifier, which can be used (in most cases) to determine why the safeguard hold is in place.\n\nTo search for a safeguard hold on a release:\n\n1. In [Windows release health](https://docs.microsoft.com/windows/release-health/), go to the **Known issues** page for a specific release (e.g. [Windows 11 known issues and notifications](https://docs.microsoft.com/windows/release-health/status-windows-11-21h2)).\n2. Search within the webpage (e.g. \"CTRL + F\") for the safeguard hold ID."
+      },
+      "name": "Title"
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "f503a201-a9bd-4085-8862-22e2c0735ed4",
+            "version": "KqlParameterItem/1.0",
+            "name": "_SnapshotTime",
+            "label": "Snapshot time",
+            "type": 1,
+            "description": "Time indicating the latest snapshot",
+            "isRequired": true,
+            "query": "UCClient\r\n| where TimeGenerated >= ago(7d)\r\n| summarize max(TimeGenerated)",
+            "isHiddenWhenLocked": true,
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          }
+        ],
+        "style": "above",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "name": "Snapshot time parameter"
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "b650e0a4-6e95-4021-b4ba-72fa7d0f7e06",
+            "version": "KqlParameterItem/1.0",
+            "name": "_SafeguardHoldID",
+            "label": "Safeguard hold ID(s)",
+            "type": 2,
+            "description": "Select a safeguard hold ID to list all devices affected by that safeguard hold.",
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "",
+            "delimiter": ",",
+            "query": "let _SafeguardHold = \"SafeguardHold\";\r\nlet _SnapshotTime = datetime(\"{_SnapshotTime}\");\r\n\r\nUCUpdateAlert\r\n| where TimeGenerated >= _SnapshotTime\r\n| where AlertSubtype == _SafeguardHold\r\n| extend SafeguardHoldIDs = parse_json(AlertData).safeguardHoldIDs\r\n| mv-expand SafeguardHoldID=SafeguardHoldIDs\r\n| summarize by SafeguardHoldID=iff(isempty(SafeguardHoldID), \"Unknown\", tostring(SafeguardHoldID))\r\n| order by SafeguardHoldID asc",
```

Comment:
do you need let _SafeguarddHold? can you just put the string directly in query?

Reply:
let's clean this up later as not breaking change